packages/e2e/features/ui/components/authenticator/useAuthenticator.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Sign Up with Username
+
+  Authenticator supports headless usage that provides access to current authenticator context
+  outside the Authenticator component. React and Vue provides it through `useAuthenticator` 
+  hook and composable respectively, and Angular provides it through `AuthenticatorService` 
+  service. They can be used to access current authState (routes), authenticated user, etc.
+
+  See https://ui.docs.amplify.aws/components/authenticator#headless for details.
+
+  Background:
+    Given I'm running the example "/ui/components/authenticator/useAuthenticator"
+
+  @angular @react @vue @todo-angular @todo-vue
```

Comment:
I can try to make sure Vue works in the future here, with this new useAuthenticator feature.

Reply:
Yep, I'll do one for Angular too. Let's do that on separate PRs though, in case there are `useAuthenticator` gotchas in other frameworks. (and tbh I'm scared I broke something this PR and have to revert it)